data/devasyarajguu.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Devasya Rajguru",
+  "bio": "Lawyer turned Hey it's me Devasya Rajguru. I have a keen interest in Web development and have a huge fascination in Open Source and technology.",
```

@devasyarajguru we can now merge your PR but are you sure you want the section `Lawyer turned`? This seems to be a copy of the template profile and you may have forgotten to delete it?
```suggestion
  "bio": "Hey it's me Devasya Rajguru. I have a keen interest in Web development and have a huge fascination in Open Source and technology.",
```